pkg/cloudclient/gcp/private.go

```diff
@@ -0,0 +1,417 @@
+package gcp
+
+//Features to add - image-id, kms-key-id
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"os"
+	"regexp"
+	"time"
+
+	"golang.org/x/oauth2/google"
+	computev1 "google.golang.org/api/compute/v1"
+
+	ocmlog "github.com/openshift-online/ocm-sdk-go/logging"
+	handledErrors "github.com/openshift/osd-network-verifier/pkg/errors"
+	"github.com/openshift/osd-network-verifier/pkg/helpers"
+	"github.com/openshift/osd-network-verifier/pkg/output"
+)
+
+type createComputeServiceInstanceInput struct {
+	ContOptImageID string
+	vpcSubnetID    string
+	userdata       string
+	// ebsKmsKeyID   string
+	zone         string
+	machineType  string
+	instanceName string
+	sourceImage  string
+	networkName  string
+}
+
+// //global variable ContOptImage image, gcp has region, zone
+
+var (
+	defaultContOptImage = map[string]string{
```

Comment:
I guess this needs to be updated, or a map isn't required

Reply:
Removed